packages/core/src/audio/AudioManager.ts

```
@@ -6,46 +6,101 @@
-      // Safari can't resume audio context without element interaction
-      document.addEventListener("pointerdown", AudioManager._tryResume, true);
-      document.addEventListener("touchend", AudioManager._tryResume, true);
-      document.addEventListener("touchstart", AudioManager._tryResume, true);
+    if (!AudioManager._context && AudioManager._hasAudio) {
+      const AudioContextClass = window.AudioContext || window.webkitAudioContext;
+
+      if (AudioContextClass) {
+        AudioManager._context = new AudioContextClass();
+
+        const originalDecodeAudioData = AudioManager._context.decodeAudioData.bind(AudioManager._context);
+        AudioManager._context.decodeAudioData = function (arrayBuffer: ArrayBuffer, successCallback?: Function, errorCallback?: Function) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)